File: apps/checker-app/code/pages/related_sops_page.py
from shiny import ui
from code.helpers.page_banner import page_banner

related_sops_page = ui.nav_panel(
    "Related SOPs",
    page_banner("Related SOPs"),
    ui.div(
        ui.tags.iframe(
            src="docs/sops/251010_SOP_template.pdf",
            width="100%",
            height="100%",
        ),
        class_="container",
        style="height: 660px;",
    ),
)

File: apps/checker-app/code/ui_pages/related_SOPs_page.R
# author: Kwame Okrah
# date: 2025-11-09

related_SOPs = tags$div(
    class="container",
    style="height: 660px",
    tags$iframe(
        src="docs/sops/251010_SOP_template.pdf",
        width="100%",
        height="100%"
    )
)

related_SOPs_page = bslib::nav_panel(
    title="Related SOPs",
    page_banner("Related SOPs"),
    related_SOPs
)